Feature/rdkemw 5868

rekhap2kandhavelan opened the pull request:
Reason for change: Native build workflow added for coverity
Priority: P1
Risks: None

balasaraswathy-n committed:
RDKEMW-5868: Native build workflow added for coverity

Reason for change: Native build workflow added for coverity

Signed-off-by: Balasaraswathy_N <Balasaraswathy_N@comcast.com>

Priority: P1
Risks: None

rkandh015 committed:
Update yml file to resolve build error

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/middleware-player-interface?pullRequest=11) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/middleware-player-interface?pullRequest=11) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: balasaraswathy-n<br/>:white_check_mark: varatharajan568<br/>:x: rkandh015<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/middleware-player-interface?pullRequest=11) it.</sub>

varatharajan568 committed:
Reason for change: Native build workflow added for coverity
Priority: P1
Risks: None

varatharajan568 committed:
Update native-full-build.yml

Resolve the script error faced in native build

varatharajan568 committed:
Update native-full-build.yml

Commented the Docker container image to resolve the native build failure faced due to using the sudo command in script files

balasaraswathy-n reviewed (approved):
Approved
